lib/cn/csched_sp3.c

```diff
@@ -1747,61 +1581,34 @@ sp3_submit(struct sp3 *sp, struct cn_compaction_work *w, uint qnum)
     sts_job_init(&w->cw_job, cn_comp_slice_cb, sp->job_id);
     sts_job_submit(sp->sts, &w->cw_job);
 
-    if (debug_sched(sp)) {
-        log_info("%-2lu %u,%-4u j%u q%u n%u t%-2u %s:%-6s  kvsets %u,%-2u  clen %5lu"
-                 "  cap %u%%  samp %lu%%%s",
-                 w->cw_tree->cnid,
-                 w->cw_node->tn_loc.node_level, w->cw_node->tn_loc.node_offset,
-                 w->cw_job.sj_id, w->cw_qnum,
-                 atomic_read(&w->cw_node->tn_busycnt) >> 16,
-                 spt->spt_job_cnt,
-                 cn_action2str(w->cw_action), cn_comp_rule2str(w->cw_comp_rule),
-                 w->cw_kvset_cnt, (uint)cn_ns_kvsets(&w->cw_ns),
-                 cn_ns_clen(&w->cw_ns) >> 20,
-                 w->cw_ns.ns_pcap,
-                 100 - (cn_ns_clen(&w->cw_ns) * 100 / cn_ns_alen(&w->cw_ns)),
-                 sp->samp_reduce ? " samp_reduce" : "");
-    }
```

Comment:
I merged this log msg with the following slog_info message.  A follow-on PR will print slog messages in plain format (with an option to print then in json format) -- and will also ensure slog messages are quiet by default.